source/operations/checklists/security.rst

```diff
@@ -8,4 +8,86 @@ Security Checklist
    :local:
    :depth: 2
 
-Stub - detail Security checklist items
+Use the following checklist when planning the security configuration for a production, distributed MinIO deployment.
+
+Required Steps
+--------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Define group policies either on MinIO or the selected 3rd party Identity Provider (LDAP/Active Directory or OpenID)
+
+   * - :octicon:`circle`
+     - Define individual access policies on MinIO or the selected 3rd party Identity Provider
+
+   * - :octicon:`circle`
+     - (Optional) Configure the tenant(s) to use the selected 3rd party Identity Provider
+
+
+Encryption-at-Rest
+------------------
+
+Specific instructions are available for various Key Management Service (KMS) provider:
+
+- :ref:`Hashicorp Vault Root KMS <>`
+- :ref:`AWS Root KMS <>`
+- :ref:`Google Cloup Platform Secret Manager Root KMS <>`
+- :ref:`Axure Key Vault Root KMS <>`
+
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Download and install the MinIO Key Encryption Service (KES)
+
+   * - :octicon:`circle`
+     - Enable TLS
+
+   * - :octicon:`circle`
+     - Generate private and public keys for KES
+
+   * - :octicon:`circle`
+     - Generate private and public keys for MinIO
+
+   * - :octicon:`circle`
+     - Create a KES configuration file and start the service
+
+   * - :octicon:`circle`
+     - Generate an external key for the key management service (KMS)
+
+   * - :octicon:`circle`
+     - Connect MinIO to the KES
+
+   * - :octicon:`circle`
+     - Enable server side encryption
+
+
+Encryption-in-Transit ("In flight")
```

Comment:
We don't have to do it now, but pre-publish we should hotlink these steps wherever we can.

Reply:
Saving as a todo.